README.adoc

```diff
@@ -209,7 +233,10 @@ Open Liberty chart that you will use to deploy your two microservices:
 helm repo add ibm-charts https://raw.githubusercontent.com/IBM/charts/master/repo/stable/
 ```
 
-A final step is required only if you are using Minikube. In this case, run the following command to configure the Docker CLI to use Minikube's Docker daemon.
+****
+[system]#*{linux}*#
+
+Run the following command to configure the Docker CLI to use Minikube's Docker daemon.
 By running this command, you will be able to interact with Minikube's Docker daemon and build new
 images directly to it from your host machine:
 
```

Comment:
Since the instructions are split up by platform, and this section is under `Linux` then we should probably only list the linux command here...